pkgs/os-specific/linux/system76-scheduler/default.nix

```diff
@@ -0,0 +1,36 @@
+{ bcc, pkg-config, dbus, lib, rustPlatform, fetchFromGitHub, tree }:
+
+let
+  version = "25a45add4300eab47ceb332b4ec07e1e74e4baaf";
```

Comment:
`version` should have the format `unstable-YYYY-MM-DD`.

Also, move this let inside the derivation.